addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -1022,27 +1022,38 @@ var SnippetsMenu = Widget.extend({
         this.customizePanel = document.createElement('div');
         this.customizePanel.classList.add('o_we_customize_panel', 'd-none');
 
-        this.invisibleDOMPanelEl = document.createElement('div');
-        this.invisibleDOMPanelEl.classList.add('o_we_invisible_el_panel');
-        this.invisibleDOMPanelEl.appendChild(
-            $('<div/>', {
-                text: _t('Invisible Elements'),
-                class: 'o_panel_header',
-            })[0]
-        );
+        if (this.options.onlyStyleTab) {
+            await this._loadSnippetsTemplates();
```

Comment:
Honestly this block is code I took over from Jabberwock. However I _do_ think it's necessary because this is where the template with the sidebar will be loaded.